webservice/webservice.py

```diff
@@ -292,7 +293,7 @@ def get_order():
 
 
 @webservicebp.route('/repository/files/export', methods=['GET'])
-def get_manifest():
+def get_manifest(format='tsv'):
     """
     Creates and returns a manifest based on the filters pased on
```

Comment:
In a previous comment, I had asked that you document the format parameter up here. Please list it, the possible values, and the default, so somebody doesn't have to go through the code.